action/protocol/poll/staking_command.go

```diff
@@ -83,22 +83,21 @@ func (sc *stakingCommand) CreatePostSystemActions(ctx context.Context, sr protoc
 }
 
 func (sc *stakingCommand) Handle(ctx context.Context, act action.Action, sm protocol.StateManager) (*action.Receipt, error) {
+	// no height here,  v1 v2 has the same validate method, so directly use common one
```

Comment:
on one hand: sc is a container, sc does/should not know the details of v1, v2

on the other hand: code should finally represent/obey to the actual situation, not bound by above

i am with the 2nd thought, this is a good comment that clearly explains the actual case, and give good readability